hack/test-templates.sh

```diff
@@ -192,7 +192,7 @@ if [ "$got" != "$expected" ]; then
 fi
 
 INFO "Testing limactl copy command"
-tmpfile="$HOME/lima-hostname"
+tmpfile="/var/tmp/lima-hostname"
```

Comment:
Not using $HOME/lima-hostname is a very good change - running tests locally should not create files in your home directory.

But this name may clash with another test, or a file created by someone else.

Best to use a temporary directory that can never clash with anything:

```
% mktemp -d /var/tmp/lima-test-templates.XXXXXX
/var/tmp/lima-test-templates.vuRmiE
```

Reply:
https://github.com/lima-vm/lima/pull/3163
I raised a pr for the change @nirs

Reply:
Thanks! but it seems that this is the only test for limactl copy.

I opened #3170 for improving limactl copy tests. I hope we can improve test coverage before we change the code to avoid regressions.

Reply:
Okay can be done after https://github.com/lima-vm/lima/pull/3163 gets merged.